src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/neoWrist/NeoWrist.java

```diff
@@ -34,6 +34,7 @@ public NeoWrist() {
                 .enableLimitSwitch(NeoWristConstants.IS_BACKWARD_LIMIT_SWITCH_ENABLED);
         motor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen)
                 .enableLimitSwitch(NeoWristConstants.IS_FORWARD_LIMIT_SWITCH_ENABLED);
+        motor.setSmartCurrentLimit(NeoWristConstants.MAX_VOLT);
```

Comment:
Isn’t current Amps and not volts?

Reply:
I have no idea it's not my code, but it's not in master so romy said I should delete it👍